### ISSUE TITLE
In nextjs Tabs component add attr sticky on error

### ISSUE THREAD

**Issue by feifaneric** [3lang3/react-vant#66]
**Describe the bug**
In Next.js I used Tabs component attr sticky on refresh browser, An error occurred 
```
ReferenceError: IntersectionObserver is not defined
```

**To Reproduce**
1. import {Tabs}
2. add sticky attr
3. refresh browser

**Screenshots**
![1](https://user-images.githubusercontent.com/52685056/133095183-5e12d4e6-54ca-468e-9683-890cc8c5f1ce.png)
![2](https://user-images.githubusercontent.com/52685056/133095189-0bf2aa28-7ee4-487e-83d2-502e4e0ad674.png)
![3](https://user-images.githubusercontent.com/52685056/133095196-400b3d09-246d-4180-9b1d-01a6a06c5c0c.png)

**Smartphone (please complete the following information):**
 - OS:win10
 - Browser chrome
 - 
**context**
```
ReferenceError: IntersectionObserver is not defined
    at useVisibilityChange (F:\my_dream\it_question\node_modules\react-vant\lib\hooks\use-visibility-change.js:22:18)
    at Sticky (F:\my_dream\it_question\node_modules\react-vant\lib\sticky\Sticky.js:173:34)
    at processChild (F:\my_dream\it_question\node_modules\react-dom\cjs\react-dom-server.node.development.js:3353:14)
    at resolve (F:\my_dream\it_question\node_modules\react-dom\cjs\react-dom-server.node.development.js:3270:5)
    at ReactDOMServerRenderer.render (F:\my_dream\it_question\node_modules\react-dom\cjs\react-dom-server.node.development.js:3753:22)
    at ReactDOMServerRenderer.read (F:\my_dream\it_question\node_modules\react-dom\cjs\react-dom-server.node.development.js:3690:29)
    at renderToString (F:\my_dream\it_question\node_modules\react-dom\cjs\react-dom-server.node.development.js:4298:27)
    at renderPage (F:\my_dream\it_question\node_modules\next\dist\next-server\server\render.js:53:854)
    at Object.ctx.renderPage (F:\my_dream\it_question\.next\server\pages\_document.js:1193:30)
    at Function.getInitialProps (F:\my_dream\it_question\.next\server\pages\_document.js:625:19)
```

**Comment by 3lang3** [comment]
fix  by e2f52da707594ed48ba1fce35f50c0c9cc4c4d0a

**Comment by 3lang3** [comment]
@all-contributors please add @feifaneric for bug
